compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/symbolic_variance.rs

```diff
@@ -54,6 +54,29 @@ impl SymbolicVariance {
         r
     }
 
+    pub fn input_external_partition(
+        nb_partitions: usize,
+        partition: PartitionIndex,
+        max_variance: f64,
+    ) -> Self {
+        let mut r = Self {
+            partition,
+            coeffs: OperationsValue::zero(nb_partitions),
+        };
+        // rust ..., offset cannot be inlined
+        *r.coeffs.input(partition) = max_variance;
```

Comment:
coeffs.input(partition) is for freshly encrypted stuff
here we should have
coeffs.pbs(partition) = 
maybe we should rename to "from_external_partition"